plugin/storage/es/spanstore/reader.go

```diff
@@ -360,9 +371,9 @@ func (s *SpanReader) FindTraceIDs(ctx context.Context, traceQuery *spanstore.Tra
 }
 
 func (s *SpanReader) multiRead(ctx context.Context, traceIDs []model.TraceID, startTime, endTime time.Time) ([]*model.Trace, error) {
-	childSpan, _ := opentracing.StartSpanFromContext(ctx, "multiRead")
-	childSpan.LogFields(otlog.Object("trace_ids", traceIDs))
-	defer childSpan.Finish()
+	ctx, childSpan := s.tracer.Start(ctx, "multiRead")
+	childSpan.SetAttributes(attribute.Key("trace_ids").String(""))
```

Comment:
Need some suggestion on how to adjust []model type here.

Reply:
Loop through trace IDs and call String() on each, and pass in a slice of strings. And before doing that you may want to check if the span is recording, since it's expensive operation.